####### The service to set the current robot position

### Request

# The current robot position to be stored.

Location position

---

### Response
